dashboard/src/shared/ResourceRef.ts

```diff
@@ -0,0 +1,43 @@
+import { Kube } from "./Kube";
+import { IResource } from "./types";
+
+class ResourceRef {
```

Comment:
I like it. I'd maybe add a comment explaining what this class means.

Reply:
done